app/views/withdraw/new.html.erb

```diff
@@ -24,33 +26,34 @@
           "contextual_guidance": "withdrawal-public-explanation-guidance",
         }
       } %>
+    </div>
 
-      <% if @edition.withdrawn? %>
-        <%= render "govuk_publishing_components/components/button", {
-          text: "Update explanation", margin_bottom: true
-        } %>
-      <% else %>
-        <%= render "govuk_publishing_components/components/button", {
-          text: "Withdraw document", margin_bottom: true
-        } %>
+    <div class="govuk-grid-column-one-third">
+      <%= render "components/contextual_guidance", {
+        id: "withdrawal-public-explanation-guidance",
+        title: t("withdraw.new.public_explanation.guidance_title"),
+      } do %>
+        <p>Explain to users why the page has been withdrawn. You can link to alternative content with markdown.</p>
+        <pre class="app-c-markdown-guidance__code-snippet" style="font-size: 12px">
```

Comment:
Thanks for flagging this, I missed it when reviewing.
I would recommend adding `word-break: break-word;` to the `.app-c-markdown-guidance__code-snippet` class to deal with text overflowing issues and preserve the font-size as is.